### ISSUE TITLE
one >53 errrors to report, reviewdog aborts GitHubActionLogWriter: "reported too many annotation"

### ISSUE THREAD

**Issue by bernhardkaindl** [tsuyoshicho/action-mypy#120]
TODO / Remaining bug:

When more than 53 errors are reported by mypy and are not filtered, reviewdog gives up and aborts with the message:

```py
GitHubActionLogWriter: "reported too many annotation"
```

The error is buried in the missle of the long output:
```py
  /home/runner/work/xen-api/xen-api/scripts/plugins/extauth-hook-AD.py:70:19: error: If x = b'abc' then f"{x}" or "{}".format(x) produces "b'abc'", not "abc". If this is desired behavior, use f"{x!r}" or "{!r}".format(x). Otherwise, decode the bytes  [str-bytes-safe]
  Warning: [mypy] reported by reviewdog 🐶
  In function "fresh_name":
  
  Raw Output:
  /home/runner/work/xen-api/xen-api/scripts/static-vdis: note: In function "fresh_name":
  reviewdog: GitHubActionLogWriter: reported too many annotation (N=53)
  + reviewdog_exit_val=1
  + echo ::endgroup::
  + [[ false == \t\r\u\e ]]
  Error: [mypy] reported by reviewdog 🐶
  Incompatible types in assignment (expression has type "str", variable has type "int")  [assignment]
  
  Raw Output:
```  

The biggest problem is that the action-mypy does not fail CI in this case, so the error goes unnoticed:

https://github.com/xenserver-next/xen-api/actions/runs/7706543512/job/21002295649

**Comment by tsuyoshicho** [comment]
Thank you, I'll try to remember it.

**Comment by tsuyoshicho** [comment]
JSON support merged
